docker-compose.yml

```diff
@@ -1,29 +1,32 @@
 version: '3.6'
 
-services:
-
-  '7.2': &base
-    image: "datadog/docker-library:ddtrace_php_7_2"
-    ulimits:
-      core: 99999999999
-    working_dir: '/home/circleci/app'
-    stdin_open: true
-    tty: true
-    volumes: [ '.:/home/circleci/app' ]
-    tmpfs: [ '/home/circleci/app/tmp:uid=3434,gid=3434,exec' ]
-    depends_on:
-      - memcached_integration
-      - redis_integration
-      - ddagent_integration
-    environment:
-      - REDIS_HOSTNAME=redis_integration
-      - DDAGENT_HOSTNAME=ddagent_integration
+x-aliases:
+  - &base_php_service
+      ulimits:
+        core: 99999999999
+      working_dir: '/home/circleci/app'
+      stdin_open: true
+      tty: true
+      volumes: [ '.:/home/circleci/app' ]
+      tmpfs: [ '/home/circleci/app/tmp:uid=3434,gid=3434,exec' ]
+      depends_on:
+        - redis_integration
```

Comment:
Just adding this comment to make sure that leaving memcached out was intentional. (It is not used though)

Reply:
Probably wrong merge, I'll re-add it